tools/depends/target/Makefile

```diff
@@ -97,6 +97,7 @@ pythonmodule-pil: $(ZLIB) libjpeg-turbo libpng freetype2 python26
 libsdl2: $(LINUX_SYSTEM_LIBS)
 libxslt: libgcrypt
 ffmpeg: $(ICONV) $(ZLIB) bzip2 libvorbis $(FFMPEG_DEPENDS)
+libcec: platform
```

Comment:
is that still needed with platform used as dependency of libcec and kodi-platform once it's adapted? I thought `DEPENDS` was just a build all target and the platform lib isn't used as a separate thing.